### PR TITLE
add positive_location schema

### PR DESCRIPTION
**Pull request by CurtisVonRubenhoff**
connects to #[issue number]

#### Reviewer don't-forgets:

- [ ] Test coverage feels appropriate, given potential risk
- [ ] We're not doubling down on already-bad code
- [ ] If there are web UI changes, they don't add anything that could be considered client-side page navigation (unless pre-approved as being necessary by another engineer)
- [ ] If there are web UI changes, they don't add any AJAX form submits (unless pre-approved as being necessary by another engineer)
- [ ] If there are any lint rules disabled, they are disabled per-line, and were (in the reviewer's judgment) appropriate to disable
- [ ] Any new environment variables used in the app are both documented and have been added to both staging and production environments already
- [ ] Potential race conditions are either inconsequential, or the code prevents them from occurring.
- [ ] If this is a UI change that called for screenshots/GIFs, in the reviewer's judgement, they were included

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -76,7 +76,8 @@ defmodule Nightingale.Mixfile do
       {:libcluster, "~> 3.0"},
       {:phoenix_live_view, "~> 0.12.0"},
       {:floki, ">= 0.0.0", only: :test},
-      {:phoenix_live_dashboard, "~> 0.1"}
+      {:phoenix_live_dashboard, "~> 0.1"},
```
Comment (CurtisVonRubenhoff) [inline mix.exs:79]: realizing i can take out these live view deps

File: lib/nightingale/positive_location.ex [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  end
+
+  defp decode_location(%{coordinates: {lng, lat}}) do
+    %{lat: lat, lng: lng}
+  end
+
+  defp decode_location(%{}) do
+    %{lat: nil, lng: nil}
+  end
+end
```
Comment (jwietelmann) [inline lib/nightingale/positive_location.ex:56]: Thanks for the Jason encoder! ...But also, please delete it. We are going to be very deliberate and versioned when we reshape the struct into JSON for any given versioned API endpont, so a single automatic encoder isn't going to cut it.

Comment (CurtisVonRubenhoff) [reply lib/nightingale/positive_location.ex:56]: roger that

**Comment by jwietelmann** [comment]
@grossvogel 

* Re: PostGIS - Ah crap
* Re: "when" - Yeah @CurtisVonRubenhoff add us a "when" field (utc_datetime)

**Pull request merged**